sklearn/ensemble/_hist_gradient_boosting/tests/test_gradient_boosting.py

```diff
@@ -66,7 +66,7 @@ def test_early_stopping_regression(scoring, validation_fraction,
 
     max_iter = 200
 
-    X, y = make_regression(random_state=0)
+    X, y = make_regression(n_samples=50, random_state=0)
```

Comment:
can these be reverted now?

Reply:
Is it a problem for this test? Generally smaller datasets make tests faster which is a good thing. Unless there would be convergence issues here?

Reply:
Agreed.

I was just wondering if we still needed them for PyPy to be happy (in which case we should be a bit more worried)

Reply:
I don't think they are needed. It's just that to revert this change now, I would have to also change CircleCI to run PyPy build here, then revert that. So I would rather avoid it, unless it's critical.